test/js/bun/console/console-iterator.test.ts

```diff
@@ -12,20 +14,19 @@ describe("should work for static input", () => {
     "1",
     "💕 Red Heart ✨ Sparkles 🔥 Fire\n💕 Red Heart ✨ Sparkles\n💕 Red Heart\n💕\n\nnormal",
   ];
-
-  for (let input of inputs) {
-    it(input.replaceAll("\n", "\\n"), () => {
-      const { stdout } = spawnSync({
-        cmd: [bunExe(), import.meta.dir + "/" + "console-iterator-run.ts"],
-        stdin: Buffer.from(input),
-        env: bunEnv,
-      });
-      expect(stdout.toString()).toBe(input.replaceAll("\n", ""));
+  async function run(input: string) {
+    const { stdout, exited } = spawn({
+      cmd: [bunExe(), dir("console-iterator-run.ts")],
+      stdin: Buffer.from(input),
+      env: bunEnv,
     });
+    expect(await new Response(stdout).text()).toBe(input.replaceAll("\n", ""));
```

Comment:
In the future this could get wonky on windows if the test file starts to contain `\r\n`.